spec/ngsild_schema.json

```diff
@@ -0,0 +1,289 @@
+                "timestamp": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "unitCode": {
+                    "type": "string"
+                }
+            }
+        },
+        "CPropertySystemData": {
+            "type": "object",
+            "required": ["dateCreated", "dateModified"],
+            "properties": {
+                "dateCreated": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "dateModified": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            }
+        },
+        "CProperty": {
+            "allOf": [
+              {
+                  "type": "object",
+                  "required": ["type", "value"],
+                  "properties": {
+                      "type": {
+                          "type": "string",
+                          "enum": ["CProperty"]
+                      },
+                      "value": {}
+                  }
+              },
+              {
+                "$ref": "#/definitions/CPropertySystemData"
+              },
+              {
+                "$ref": "#/definitions/CPropertyCommonData"
+              }
+            ]
+        },
+
+
+        "CRelationshipCommonData": {
+            "type": "object",
+            "properties": {
+                "timestamp": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            }
+        },
+        "CRelationshipSystemData": {
+            "type": "object",
+            "properties": {
+                "dateCreated": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "dateModified": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            }
+        },
+        "CRelationship": {
+            "allOf": [
+                {
+                    "type": "object",
+                    "required": ["type", "object"],
+                    "properties": {
+                        "type": {
+                            "type": "string",
+                            "enum": ["CRelationship"]
+                        },
+                        "object": {
+                          "type": "string",
+                          "format": "uri"
+                        }
+                    }
+                },
+                {
+                    "$ref": "#/definitions/CRelationshipSystemData"
+                },
+                {
+                    "$ref": "#/definitions/CRelationshipCommonData"
+                }
+            ]
+        },
+
+
+        "EntityInfo": {
+            "type": "object",
+            "required": ["id"],
+            "properties": {
+                "id": {
+                    "type": "string"
```

Comment:
uri

Reply:
Not sure if 'uri' is allowed by the parser as a type. I had pending check if possible set type as string and include 'format: uri'.